added schema for ifsc

SonalBhanawat opened the pull request.

jophish commented on fiatconnect-api.md:
```diff
@@ -2223,6 +2224,25 @@ platform can be used to consume Fiat Connect services by providing their `mobile
+{
+	accountName: `string`,
+	institutionName: `string`,
+	ifsc: `string`,
```
What is a client supposed to provide for the `ifsc` field? It seems like the IFSC number ought to be provided in the `accountNumber` field -- what is the `ifsc` field for?

jophish commented on fiatconnect-api.md:
```diff
@@ -2223,6 +2224,25 @@ platform can be used to consume Fiat Connect services by providing their `mobile
+	institutionName: `string`,
+	ifsc: `string`,
+	accountNumber: `string`,
+	country: `string`,
```
Are IFSC-based accounts available anywhere _other_ than India? Will a user ever be able to set this to anything other than `"IND"`/will the `allowedValues` field allow the user to do so?

msshahanshah replied:
IFCS is the branch identifier used in India. 
`RATN0000012` is a sample. This string has to be known before a transaction to the account number can be initiated. 
IFSC is only applicable for INR transactions.

jophish commented on fiatconnect-api.md:
```diff
@@ -2223,6 +2224,25 @@ platform can be used to consume Fiat Connect services by providing their `mobile
+	accountName: `string`,
+	institutionName: `string`,
+	ifsc: `string`,
+	accountNumber: `string`,
```
Is there a general syntax required for `accountNumber` or does it vary from bank to bank? If it's the same syntax for all banks, could you please specify it in the paragraph below?

jophish reviewed (approved):
This looks good to me; just one small comment about the syntax for `accountNumber`!

jophish commented on fiatconnect-api.md:
```diff
@@ -2245,6 +2246,25 @@ platform can be used to consume Fiat Connect services by providing their `mobile
+	ifsc: `string`,
+	accountNumber: `string`,
+	country: `string`,
+	fiatAccountType: `FiatAccountTypeEnum.IfscAccount`
```
@SonalBhanawat This schema represents a bank account; I think we should use `FiatAccountTypeEnum.BankAccount` here instead.

jophish commented:
@SonalBhanawat Just made some minor updates to clean some things up; in particular, I changed the schema name to `IFSCAccount` (uppercased the whole `Ifsc` bit). I also changed the `fiatAccountType` field to `BankAccount`, since this schema is used to represent bank accounts; no need for a brand new `FiatAccountType` for this. Let me know that this sounds good and I'll merge.

SonalBhanawat commented:
> @SonalBhanawat Just made some minor updates to clean some things up; in particular, I changed the schema name to `IFSCAccount` (uppercased the whole `Ifsc` bit). I also changed the `fiatAccountType` field to `BankAccount`, since this schema is used to represent bank accounts; no need for a brand new `FiatAccountType` for this. Let me know that this sounds good and I'll merge.

@jophish - Looks fine.Please merge.
